app/models/user.rb

```diff
@@ -1,2 +1,10 @@
 class User < ApplicationRecord
+  before_save { self.email = email.downcase }
+  validates :name,  presence: true, length: { maximum: 50 }
```

Comment:
Trong model nhóm thành 1 cụm nhé ví dụ:

VALID_EMAIL_REGEX

toàn bộ validates

has_secure_password